src/client/telemetry/index.ts

```diff
@@ -911,6 +911,24 @@ export interface IEventNamePropertyMapping {
          */
         disabled: boolean;
     };
+    /**
+     * Telemetry event sent when 'Select Interpreter' command is invoked
+     */
+    [EventName.SELECT_INTERPRETER]: never | undefined;
+    /**
+     * Telemetry event sent when 'Enter interpreter path' button is clicked
+     */
+    [EventName.ENTER_INTERPRETER_PATH_BUTTON]: never | undefined;
+    /**
+     * Telemetry event sent with details about what choice user made to input the interpreter path
+     */
+    [EventName.INPUT_INTERPRETER_PATH_CHOICES]: {
+        /**
+         * Carries 'enter' if user chose to enter the path to executable
+         * Carries 'browse' if user chose to browse for the path to the executable
```

Comment:
Don't forget to end sentences with a period, even in basic comments like this:

```suggestion
     * Telemetry event sent when 'Select Interpreter' command is invoked.
     */
    [EventName.SELECT_INTERPRETER]: never | undefined;
    /**
     * Telemetry event sent when 'Enter interpreter path' button is clicked.
     */
    [EventName.ENTER_INTERPRETER_PATH_BUTTON]: never | undefined;
    /**
     * Telemetry event sent with details about what choice user made to input the interpreter path.
     */
    [EventName.INPUT_INTERPRETER_PATH_CHOICES]: {
        /**
         * Carries 'enter' if user chose to enter the path to executable.
         * Carries 'browse' if user chose to browse for the path to the executable.
```

Reply:
It wasn't done in any of the places in that file, but we can start now.